cobalt/browser/application.cc

```diff
@@ -732,19 +731,8 @@ Application::Application(const base::Closure& quit_closure, bool should_preload,
     options.renderer_module_options.enable_fps_overlay = true;
   }
 
-  if (command_line->HasSwitch(browser::switches::kEnableSkiaRasterizer)) {
-    int enable_skia = 0;
-    base::StringToInt(command_line->GetSwitchValueASCII(
-                          browser::switches::kEnableSkiaRasterizer),
-                      &enable_skia);
-    if (enable_skia) {
-      options.renderer_module_options.rasterizer_type_setting =
-          configuration::Configuration::kSkiaRasterizer;
-    } else {
-      options.renderer_module_options.rasterizer_type_setting =
-          configuration::Configuration::kGlesRasterizer;
-    }
-  }
+  options.renderer_module_options.rasterizer_type_setting =
+      browser::GetRasterizerType();
```

Comment:
Does GetRasterizerType need to be it's own resolve_rasterizer_settings.cc file? I'm fine either way, it's nice to not have to duplicate code, but it could be something like (pseudo)

base::Value value;
persistent_settings_->Get(kSkiaRasterizer, &value);
if (value) {
  optionsrenderer_module_options.rasterizer_type_setting = configuration::Configuration::kSkiaRasterizer;
} else {
  optionsrenderer_module_options.rasterizer_type_setting = configuration::Configuration::kGlesRasterizer;
}

Reply:
We want to have that command line switch in the function as well, so probably we don't want to duplicate all these code in 4 different places. See above Kaido's comments.